gitlab-status/handler.go

```diff
@@ -88,8 +90,8 @@ func sendReport(URL string, token string, state string, desc string, context str
 		return fmt.Errorf("error while creating request to GitLab API: %s", reqErr.Error())
 	}
 	req.Header.Set("PRIVATE-TOKEN", token)
-	client := &http.Client{}
```

Comment:
Can you use `http.DefaultClient` instead?

Reply:
Done